bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/filters/TestTimestampsFilterAdapter.java

```diff
@@ -36,18 +36,18 @@
   public void timestampFiltersAreAdapted() {
     // Timestamps are sorted by the filter min -> max
     TimestampsFilter filter = new TimestampsFilter(ImmutableList.of(1L, 10L, 20L));
-    RowFilter rowFilter = filterAdapter.adapt(emptyScanContext, filter);
-    Assert.assertEquals(3, rowFilter.getInterleave().getFiltersCount());
+    Filters.Filter expected = filterAdapter.adapt(emptyScanContext, filter);
+    Assert.assertEquals(3, expected.toProto().getInterleave().getFiltersCount());
     Assert.assertEquals(
         10000L,
-        rowFilter
+        expected.toProto()
```

Comment:
Please revert `expected` to a `RowFilter`, since there are 3 conversions with `.toProto()`

Reply:
instead of reverting to `RowFilter`, extracted a variable for `expected.toProto().getInterleave()`, [here](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/pull/1991/files#diff-c3456ef2c51a68d549677d84c02af190R40)